Formula/netdata.rb

```diff
@@ -53,25 +49,10 @@ def install
     # https://github.com/protocolbuffers/protobuf/issues/9947
     ENV.append_to_cflags "-DNDEBUG"
 
-    # We build judy as static library, so we don't need to install it
-    # into the real prefix
-    judyprefix = "#{buildpath}/resources/judy"
-
-    resource("judy").stage do
-      system "./configure", "--disable-debug", "--disable-dependency-tracking",
-          "--disable-shared", "--prefix=#{judyprefix}"
-
-      # Parallel build is broken
-      ENV.deparallelize do
-        system "make", "-j1", "install"
-      end
-    end
-
     ENV["PREFIX"] = prefix
-    ENV.append "CFLAGS", "-I#{judyprefix}/include"
-    ENV.append "LDFLAGS", "-L#{judyprefix}/lib"
```

Comment:
These might still be needed because `judy` doesn't ship a `.pc` file, except with `judyprefix = Formula["judy"].opt_prefix`.

But let's see if it works without.

Reply:
looks like it works fine.

Reply:
I don't think so, since there's no linkage with `judy`. Try adding this to see if we get linkage.